testing/e2e/manifests/operator.go

```diff
@@ -39,19 +39,19 @@ var (
 type OperatorVersion uint
 
 const (
-	OperatorVersion0_0_3 OperatorVersion = iota // use iota to number with earlier versions being lower numbers
-	OperatorVersion0_2_0
+	OperatorVersion0_2_1_Patch_4 OperatorVersion = iota // use iota to number with earlier versions being lower numbers
+	OperatorVersion0_2_3_Patch_2
 
 	// OperatorVersionLatest represents the latest version of the operator which is essentially whatever code changes this test is running against
 	OperatorVersionLatest = math.MaxUint // this must always be the last/largest value in the enum because we order by value
 )
 
 func (o OperatorVersion) String() string {
 	switch o {
-	case OperatorVersion0_0_3:
-		return "0.0.3"
-	case OperatorVersion0_2_0:
-		return "0.2.0"
+	case OperatorVersion0_2_1_Patch_4:
```

Comment:
approving but might be worth looking into a way to automate which versions are used so that we don't have to update them every time we release a new patch/a new version we need to test.

Reply:
probably some way to utilize the `active_releases.json` file I added last week